diff --git a/beeond/CMakeLists.txt b/beeond/CMakeLists.txt
index 59b5a58..fc852ea 100644
--- a/beeond/CMakeLists.txt
+++ b/beeond/CMakeLists.txt
@@ -1,11 +1,11 @@
 install(
-	PROGRAMS "beegfs-ondemand-stoplocal" "beeond-lib"
-	DESTINATION "usr/share/beeond"
+	PROGRAMS "scripts/lib/beegfs-ondemand-stoplocal" "scripts/lib/beeond-lib"
+	DESTINATION "share/beeond"
 	COMPONENT "beeond"
 )
 
 install(
-	PROGRAMS "beeond" "beeond-cp"
-	DESTINATION "usr/bin"
+	PROGRAMS "source/beeond" "source/beeond-cp"
+	DESTINATION "bin"
 	COMPONENT "beeond"
 )
diff --git a/beeond_thirdparty_gpl/CMakeLists.txt b/beeond_thirdparty_gpl/CMakeLists.txt
index 0875e41..9ae3a34 100644
--- a/beeond_thirdparty_gpl/CMakeLists.txt
+++ b/beeond_thirdparty_gpl/CMakeLists.txt
@@ -12,12 +12,12 @@ ExternalProject_Add(
 
 install(
 	PROGRAMS "${CMAKE_CURRENT_BINARY_DIR}/parallel-prefix/src/parallel-${parallel_version}/src/parallel"
-	DESTINATION "usr/share/beeond/parallel"
+	DESTINATION "share/beeond/parallel"
 	COMPONENT "beeond-thirdparty-gpl"
 )
 
 install(
 	FILES "${CMAKE_CURRENT_BINARY_DIR}/parallel-prefix/src/parallel-${parallel_version}/COPYING"
-	DESTINATION "usr/share/beeond/parallel"
+	DESTINATION "share/beeond/parallel"
 	COMPONENT "beeond-thirdparty-gpl"
 )
diff --git a/client_devel/CMakeLists.txt b/client_devel/CMakeLists.txt
index 7aafeed..b66080d 100644
--- a/client_devel/CMakeLists.txt
+++ b/client_devel/CMakeLists.txt
@@ -1,11 +1,11 @@
 install(
 	FILES "include/beegfs/beegfs.h" "include/beegfs/beegfs_ioctl.h" "include/beegfs/beegfs_ioctl_functions.h"
-	DESTINATION "usr/include/beegfs"
+	DESTINATION "include/beegfs"
 	COMPONENT "client-devel"
 )
 
 install(
 	FILES "build/dist/usr/share/doc/beegfs-client-devel/examples/createFileWithStripePattern.cpp"
-	DESTINATION "usr/share/doc/beegfs/examples/createFileWithStripePattern"
+	DESTINATION "share/doc/beegfs/examples/createFileWithStripePattern"
 	COMPONENT "client-devel"
 )
diff --git a/client_module/CMakeLists.txt b/client_module/CMakeLists.txt
index c697f1e..527d343 100644
--- a/client_module/CMakeLists.txt
+++ b/client_module/CMakeLists.txt
@@ -34,7 +34,7 @@ configure_file(
 
 install(
 	DIRECTORY ""
-	DESTINATION "usr/src/beegfs-${BEEGFS_VERSION}"
+	DESTINATION "src/beegfs-${BEEGFS_VERSION}"
 	COMPONENT "client"
 	USE_SOURCE_PERMISSIONS
 	PATTERN "CMakeLists.txt" EXCLUDE
@@ -43,7 +43,7 @@ install(
 
 install(
 	FILES "${CMAKE_CURRENT_BINARY_DIR}/dkms.conf.client"
-	DESTINATION "usr/src/beegfs-${BEEGFS_VERSION}"
+	DESTINATION "src/beegfs-${BEEGFS_VERSION}"
 	RENAME "dkms.conf"
 	COMPONENT "client"
 )
@@ -56,7 +56,7 @@ install(
 
 install(
 	DIRECTORY ""
-	DESTINATION "usr/src/beegfs-compat-${BEEGFS_VERSION}"
+	DESTINATION "src/beegfs-compat-${BEEGFS_VERSION}"
 	COMPONENT "client-compat"
 	USE_SOURCE_PERMISSIONS
 	PATTERN "CMakeLists.txt" EXCLUDE
@@ -65,7 +65,7 @@ install(
 
 install(
 	FILES "${CMAKE_CURRENT_BINARY_DIR}/dkms.conf.compat"
-	DESTINATION "usr/src/beegfs-compat-${BEEGFS_VERSION}"
+	DESTINATION "src/beegfs-compat-${BEEGFS_VERSION}"
 	RENAME "dkms.conf"
 	COMPONENT "client-compat"
 )
diff --git a/common/CMakeLists.txt b/common/CMakeLists.txt
index 9c15aed..184f38e 100644
--- a/common/CMakeLists.txt
+++ b/common/CMakeLists.txt
@@ -572,6 +572,6 @@ target_link_libraries(
 
 install(
 	TARGETS beegfs_ib
-	DESTINATION "usr/lib"
+	DESTINATION "lib"
 	COMPONENT "libbeegfs-ib"
 )
diff --git a/ctl/CMakeLists.txt b/ctl/CMakeLists.txt
index 56fd51b..0075bc9 100644
--- a/ctl/CMakeLists.txt
+++ b/ctl/CMakeLists.txt
@@ -149,6 +149,6 @@ endif()
 
 install(
 	TARGETS beegfs-ctl
-	DESTINATION "usr/bin"
+	DESTINATION "bin"
 	COMPONENT "utils"
 )
diff --git a/event_listener/CMakeLists.txt b/event_listener/CMakeLists.txt
index 31764fd..ede40bf 100644
--- a/event_listener/CMakeLists.txt
+++ b/event_listener/CMakeLists.txt
@@ -10,13 +10,13 @@ add_executable(
 
 install(
 	TARGETS beegfs-event-listener
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "event-listener"
 )
 
 install(
 	FILES "include/beegfs/beegfs_file_event_log.hpp"
-	DESTINATION "usr/include/beegfs"
+	DESTINATION "include/beegfs"
 	COMPONENT "event-listener"
 )
 
diff --git a/helperd/CMakeLists.txt b/helperd/CMakeLists.txt
index b3c4a34..9ceb468 100644
--- a/helperd/CMakeLists.txt
+++ b/helperd/CMakeLists.txt
@@ -25,6 +25,7 @@ target_link_libraries(
 	helperd
 	beegfs-common
 	pthread
+  dl
 )
 
 add_executable(
@@ -58,6 +59,6 @@ endif()
 
 install(
 	TARGETS beegfs-helperd
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "helperd"
 )
diff --git a/meta/CMakeLists.txt b/meta/CMakeLists.txt
index 8907f21..45db4c7 100644
--- a/meta/CMakeLists.txt
+++ b/meta/CMakeLists.txt
@@ -326,19 +326,19 @@ endif()
 
 install(
 	TARGETS beegfs-meta
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "meta"
 )
 
 install(
 	PROGRAMS "build/dist/sbin/beegfs-setup-meta"
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "meta"
 )
 
 install(
 	FILES "build/dist/usr/lib/systemd/system/beegfs-meta.service" "build/dist/usr/lib/systemd/system/beegfs-meta@.service"
-	DESTINATION "${CMAKE_INSTALL_LIBDIR}/systemd/system"
+	DESTINATION "lib/systemd/system"
 	COMPONENT "meta"
 )
 
@@ -348,9 +348,3 @@ install(
 	COMPONENT "meta"
 )
 
-install(
-	PROGRAMS "build/beegfs-meta.sh"
-	RENAME "beegfs-meta"
-	DESTINATION "opt/beegfs/sbin"
-	COMPONENT "meta"
-)
diff --git a/mgmtd/CMakeLists.txt b/mgmtd/CMakeLists.txt
index 16d5732..0214a3d 100644
--- a/mgmtd/CMakeLists.txt
+++ b/mgmtd/CMakeLists.txt
@@ -161,19 +161,19 @@ endif()
 
 install(
 	TARGETS beegfs-mgmtd
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "mgmtd"
 )
 
 install(
 	PROGRAMS "build/dist/sbin/beegfs-setup-mgmtd"
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "mgmtd"
 )
 
 install(
 	FILES "build/dist/usr/lib/systemd/system/beegfs-mgmtd.service" "build/dist/usr/lib/systemd/system/beegfs-mgmtd@.service"
-	DESTINATION "${CMAKE_INSTALL_LIBDIR}/systemd/system"
+	DESTINATION "lib/systemd/system"
 	COMPONENT "mgmtd"
 )
 
@@ -183,9 +183,3 @@ install(
 	COMPONENT "mgmtd"
 )
 
-install(
-	PROGRAMS "build/beegfs-mgmtd.sh"
-	RENAME "beegfs-mgmtd"
-	DESTINATION "opt/beegfs/sbin"
-	COMPONENT "mgmtd"
-)
diff --git a/mon/CMakeLists.txt b/mon/CMakeLists.txt
index c68b1a5..46333f0 100644
--- a/mon/CMakeLists.txt
+++ b/mon/CMakeLists.txt
@@ -51,7 +51,7 @@ add_library(
 
 target_include_directories(
 	mon PRIVATE
-	../thirdparty/datastax
+	../thirdparty/source/datastax
 )
 
 target_link_libraries(
@@ -98,13 +98,13 @@ target_link_libraries(
 
 install(
 	TARGETS beegfs-mon
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "mon"
 )
 
 install(
 	FILES "build/dist/usr/lib/systemd/system/beegfs-mon.service" "build/dist/usr/lib/systemd/system/beegfs-mon@.service"
-	DESTINATION "${CMAKE_INSTALL_LIBDIR}/systemd/system"
+	DESTINATION "lib/systemd/system"
 	COMPONENT "mon"
 )
 
diff --git a/storage/CMakeLists.txt b/storage/CMakeLists.txt
index 55b8f15..a049c6c 100644
--- a/storage/CMakeLists.txt
+++ b/storage/CMakeLists.txt
@@ -188,19 +188,19 @@ endif()
 
 install(
 	TARGETS beegfs-storage
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "storage"
 )
 
 install(
 	FILES "build/dist/usr/lib/systemd/system/beegfs-storage.service" "build/dist/usr/lib/systemd/system/beegfs-storage@.service"
-	DESTINATION "${CMAKE_INSTALL_LIBDIR}/systemd/system"
+	DESTINATION "lib/systemd/system"
 	COMPONENT "storage"
 )
 
 install(
 	PROGRAMS "build/dist/sbin/beegfs-setup-storage"
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "storage"
 )
 
@@ -210,9 +210,3 @@ install(
 	COMPONENT "storage"
 )
 
-install(
-	PROGRAMS "build/beegfs-storage.sh"
-	RENAME "beegfs-storage"
-	DESTINATION "opt/beegfs/sbin"
-	COMPONENT "storage"
-)
diff --git a/upgrade/beegfs_mirror_md/CMakeLists.txt b/upgrade/beegfs_mirror_md/CMakeLists.txt
index a0046fe..c90ceef 100644
--- a/upgrade/beegfs_mirror_md/CMakeLists.txt
+++ b/upgrade/beegfs_mirror_md/CMakeLists.txt
@@ -15,6 +15,6 @@ target_link_libraries(
 
 install(
 	TARGETS "beegfs-mirror-md"
-	DESTINATION "usr/sbin"
+	DESTINATION "sbin"
 	COMPONENT "beegfs-mirror-md"
 )
diff --git a/utils/CMakeLists.txt b/utils/CMakeLists.txt
index 88d98c0..9968642 100644
--- a/utils/CMakeLists.txt
+++ b/utils/CMakeLists.txt
@@ -6,6 +6,6 @@ install(
 
 install(
 	PROGRAMS "scripts/beegfs-check-servers" "scripts/beegfs-df" "scripts/beegfs-net"
-	DESTINATION "usr/bin"
+	DESTINATION "bin"
 	COMPONENT "utils"
 )
