diff --git a/nhc b/nhc
index 1705e79..ca8d9d5 100755
--- a/nhc
+++ b/nhc
@@ -180,7 +180,6 @@ function nhcmain_init_env() {
     ### Variable declarations
 
     # Static variables
-    PATH="/sbin:/usr/sbin:/bin:/usr/bin"
     SYSCONFIGDIR="/etc/sysconfig"
     LIBEXECDIR="/usr/libexec"
     if [[ -r /proc/sys/kernel/hostname ]]; then
diff --git a/nhc-genconf b/nhc-genconf
index 5483465..0e63ac4 100755
--- a/nhc-genconf
+++ b/nhc-genconf
@@ -55,7 +55,6 @@ function dbg() {
 
 function nhcgc_init_env() {
     umask 0077
-    PATH="/sbin:/usr/sbin:/bin:/usr/bin"
     SYSCONFIGDIR="/etc/sysconfig"
     LIBEXECDIR="/usr/libexec"
     if [[ -r /proc/sys/kernel/hostname ]]; then
@@ -316,7 +315,7 @@ function nhcgc_gen_config() {
                 fi
             done
         fi
-    fi        
+    fi
 
     # Hardware
     dbg "Generating hardware section."
@@ -339,12 +338,12 @@ function nhcgc_gen_config() {
         for ((i=0; i<${#HW_IB_DEV[*]}; i++)); do
             echo " $HOSTNAME || check_hw_ib ${HW_IB_RATE[$i]} ${HW_IB_DEV[$i]}"
         done
-    fi        
+    fi
     if [[ ${#HW_ETH_DEV[*]} -gt 0 ]]; then
         for ((i=0; i<${#HW_ETH_DEV[*]}; i++)); do
             echo " $HOSTNAME || check_hw_eth ${HW_ETH_DEV[$i]}"
         done
-    fi        
+    fi
 
     # nVidia GPUs
     if [[ $NV_HEALTHMON_RC -eq 0 ]]; then
@@ -356,7 +355,7 @@ function nhcgc_gen_config() {
         echo "### nVidia GPU checks"
         echo "###"
         echo " $HOSTNAME || check_nv_healthmon"
-    fi        
+    fi
 
     # Running processes and load average
     if [[ ${#PS_PROCS[*]} -gt 0 ]]; then
@@ -414,7 +413,7 @@ function nhcgc_gen_config() {
                 fi
             fi
         done
-    fi        
+    fi
 
 }
 
diff --git a/nhc-wrapper b/nhc-wrapper
index 1ee4664..4eeaffd 100755
--- a/nhc-wrapper
+++ b/nhc-wrapper
@@ -83,9 +83,7 @@ function nhc_parse_timespec() {
 
 function nhcwrap_init_env() {
     umask 0077
-    PATH="/sbin:/usr/sbin:/bin:/usr/bin"
     SYSCONFIGDIR="/etc/sysconfig"
-    LIBEXECDIR="/usr/libexec"
     if [[ -r /proc/sys/kernel/hostname ]]; then
         read HOSTNAME < /proc/sys/kernel/hostname
     else
